src/browser/migration-layer/is-using-latest.js

```diff
@@ -11,8 +11,8 @@ function isUsingLatest(buildName) {
   var scripts = document.getElementsByTagName('script');
   var found = false;
   for (var currentScript = 0, nbScripts = scripts.length;
-        currentScript < nbScripts;
-        currentScript++) {
+    currentScript < nbScripts;
```

Comment:
Don't you think it's a bit confusing to have the for parts with the same indentation as the inner code?